TreeDB/cmd/treedb_rag_benchmark/cell_worker.go

```
@@ -0,0 +1,201 @@
+		}
+		attempt := attempts[request.Ordinal]
+		attempts[request.Ordinal] = attempt + 1
+
+		var state *applicationCellWorkerEnvironment
+		var env *applicationEnvironment
+		var queryVectors map[string][]float32
+		if unsupportedCapability(cell) == nil {
+			state, err := openEnvironment(cell.Embedding, request.Ordinal, attempt)
+			if err != nil {
+				response := applicationCellWorkerResponse{Ordinal: request.Ordinal, Error: err.Error()}
+				_ = encoder.Encode(response)
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked: unsafe type `float64` found

(errchkjson)